src/NServiceBus.AcceptanceTests/Serialization/When_configuring_custom_xml_namespace.cs

```diff
@@ -0,0 +1,83 @@
+﻿namespace NServiceBus.AcceptanceTests.Serialization
+{
+    using System.Text;
+    using System.Threading.Tasks;
+    using System.Xml.Linq;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using MessageMutator;
+    using NUnit.Framework;
+
+    public class When_configuring_custom_xml_namespace : NServiceBusAcceptanceTest
+    {
+        const string CustomXmlNamespace = "http://particular.net";
```

Comment:
Just by looking at this constant I would have assumed that this is the default namespace :)